code/images/SCIENCE.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<!-- saved from url=(0037)http://127.0.0.1:5500/code/index.html -->
+<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    <title> SCIENCE </title>
+    
+    
+    
+    <!-- Bootstrap link -->
+    
+    <!-- Fontawesome link -->
+    
+    
+  <link rel="shortcut icon" href="http://127.0.0.1:5500/code/favicon.ico" type="image/x-icon"><link rel="stylesheet" href="./SCIENCE_files/style.css"><link href="./SCIENCE_files/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous"><link rel="stylesheet" href="./SCIENCE_files/all.min.css"></head>
+  <body>
+    <header>
+      <div class="nav-bar nav-bar-expand">
+      
+        <img src="./SCIENCE_files/logo.png" alt="logo">
+        
+        <h1>SCIENCE WORLD</h1>
+        
+        
+    </div>
+    <!-- Navbar links -->
+    <nav class="navbar navbar-expand">
+          
+              <ul class="navbar-nav ">
+                <li class="nav-item">
+                  <a class="nav-link" href="http://127.0.0.1:5500/code/index.html#" target="_blank">HOME</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="http://127.0.0.1:5500/code/index.html#" target="_blank">EARTH</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="http://127.0.0.1:5500/code/index.html#" target="_blank">SPACE</a>
+                </li> 
+              <li class="nav-item">
+                  <a class="nav-link" href="http://127.0.0.1:5500/code/index.html#" target="_blank">NATURE</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="http://127.0.0.1:5500/code/index.html#" target="_blank">EVOLUTION</a>
+                </li>
+               
+              <li class="nav-item">
+                  <a class="nav-link" href="http://127.0.0.1:5500/code/index.html#" target="_blank">VIDEOS</a>
+                </li>
+                
+                
+              </ul>
+              <div class="media-icons"> 
+        
+                <i class="fab fa-facebook-f"></i>
+          <i class="fab fa-instagram"></i>
+                <i class="fab fa-twitter"></i>
+                <i class="fab fa-youtube"></i>
+            </div> 
+
+              
+
+             
+          </nav>
+          <div class="highlights">
+            <p class="highlights1">INTERESTING FACTS</p>
+            <!--The <marquee> HTML element is used to insert a scrolling area of text. -->
+            <marquee direction="left">
+              <p class="highlights2">Hot water freezes faster than cold water. 
+               
+               </p>
+            </marquee>
+          
+          </div>
+ 
+    </header>
+     
+    <main class="main-news">
```

Comment:
I like how you're naming the classes, makes it easier for me to know where I am in the code. Good job